### PR TITLE
NAS-116395 / 22.12 / NAS-116395: Pool usage details widget/card

### PR DESCRIPTION
**Pull request by bvasilenko**
Testing:
Temporary location of this widget is Storage2 page

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -116,6 +116,7 @@
     "moment-timezone": "~0.5.33",
     "ng-chartist": "~4.1.0",
     "ng-mocks": "~13.5.2",
+    "ng2-charts": "~2.4.1",
```
Comment (denysbutenko) [inline package.json:119]: We already have d3 and probably other libraries for charts, what is benefit of this one?

File: src/app/pages/storage2/components/pools-dashboard/widget-usage/widget-usage.component.ts [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+interface UsageState {
+  capacity: string;
+  used: string;
+  avail: string;
+  usedPct: number;
+  health: UsageHealthLevel;
+}
+
+export enum UsageHealthLevel {
+  Warn = 'warn',
```
Comment (denysbutenko) [inline src/app/pages/storage2/components/pools-dashboard/widget-usage/widget-usage.component.ts:22]: Keys don't need shortcuts

File: src/app/pages/storage2/components/pools-dashboard/widget-usage/widget-usage.component.ts [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+const unknownSize = 'Unknown';
+const maxPct = 80;
+
+@UntilDestroy()
+@Component({
+  selector: 'ix-widget-usage',
+  templateUrl: './widget-usage.component.html',
+  styleUrls: [
+    './widget-usage.component.scss',
+    '../../../../dashboard/components/widget/widget.component.scss',
```
Comment (denysbutenko) [inline src/app/pages/storage2/components/pools-dashboard/widget-usage/widget-usage.component.ts:36]: Perhaps can be replaced with absolute path

File: src/app/pages/storage2/components/pools-dashboard/pools-dashboard.component.html [hunk 4]
```diff
@@ -3,4 +3,16 @@
         <h4>Note: This is a work in progress for new 'Pools Dashboard' page. You can see what we are building an provide suggestions in <a href="https://github.com/truenas/webui/discussions/6767">Github discussions</a>.</h4>
     </div></mat-card-content></mat-card>
   <button mat-button (click)="navigateToDeviceManagement()"> {{ 'Manage Devices' | translate }}</button>
+  <div class="spinner-container" *ngIf="isLoading">
```
Comment (denysbutenko) [inline src/app/pages/storage2/components/pools-dashboard/pools-dashboard.component.html:6]: There is new approach for loading state, look for ngx-skeleton-loader examples

**Review by denysbutenko — COMMENTED** [review]
I'll test it and provide more feedback

**Review by denysbutenko — CHANGES_REQUESTED** [review]
image
Does not follow design according to [discussions#6767](https://github.com/truenas/webui/discussions/6767) and other widgets

name

**Comment by bvasilenko** [comment]
@denysbutenko 
I've done the improvements according to the feedback. Could you please test one more time?

**Review by denysbutenko — APPROVED** [review]
image

Looks good 👍🏻

**Pull request merged**